.github/workflows/test-sim.yml

```diff
@@ -23,9 +23,9 @@ on:
         default: 40
 
 env:
-  GETH_DOCKER_IMAGE: ethereum/client-go:v1.11.6
+  GETH_DOCKER_IMAGE: ethereum/client-go:v1.13.11
   LIGHTHOUSE_DOCKER_IMAGE: sigp/lighthouse:latest-amd64-modern-dev
```

Comment:
we should use a release version for lighthouse as well

Reply:
Yes the build with `-dev` is the right one. Seems for latest releases they started the dev builds. I will update it shortly.